tower-batch/tests/ed25519.rs

```diff
@@ -69,53 +77,75 @@ impl<'msg> Service<BatchControl<Request<'msg>>> for Ed25519Verifier {
 impl Drop for Ed25519Verifier {
     fn drop(&mut self) {
         // We need to flush the current batch in case there are still any pending futures.
-        let batch = std::mem::replace(&mut self.batch, BatchVerifier::default());
+        let batch = std::mem::replace(&mut self.batch, batch::Verifier::default());
         let _ = self.tx.send(batch.verify(thread_rng()));
     }
 }
 
 // =============== testing code ========
 
+static LOGGER_INIT: Once = Once::new();
+
+fn install_tracing() {
```

Comment:
we should probably move this code to some shared test crate so it doesn't get duplicated all over our codebase.